calcom/cal.com
# fix: App Install Dropdown Sort Properly [CAL-2285]

## What does this PR do?



This PR fixes the app install dropdown menu to show data in the correct order of:
- User
- Org
- Teams

This PR also fixes the placeholder for users that have not completed onboarding.

![CleanShot 2023-08-09 at 13 31 15](https://github.com/calcom/cal.com/assets/65426560/02d4efe2-77ca-4c7e-a96c-e38bc712b620)


Fixes #10542



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change





## How should this be tested?



- Create an org with multiple teams
- Do not complete onboarding for the org admin or delete the name in the DB
- Go to the app install page
- It should look similar to the screenshot attached

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
